python/ray/runtime_context.py

```diff
@@ -251,7 +251,7 @@ def get_actor_id(self) -> Optional[str]:
         """
         # only worker mode has actor_id
         if self.worker.mode != ray._private.worker.WORKER_MODE:
-            logger.warning(
+            logger.debug(
```

Comment:
Also changed this to DEBUG because it's very spammy when the driver calls this method. It seems fine to me if the driver calls it, not sure why this should be a warning.

Reply:
Looks good, thanks!